.ci/linux-steps.yaml

```diff
@@ -23,16 +23,11 @@ steps:
 
     sudo apt-get install -y --allow-unauthenticated libopenblas-dev liblapack-dev g++ libboost1.70-dev libarmadillo-dev xz-utils
 
-    if [ '$(python.version)' == '2.7' ]; then
-      sudo apt-get install -y --allow-unauthenticated python-pip cython python-numpy python-pandas
-      sudo pip install --upgrade --ignore-installed setuptools cython
-      sudo pip install "zipp<3.1"
-    fi
-
-    if [ '$(python.version)' == '3.7' ]; then
-      sudo apt-get install -y --allow-unauthenticated python3-pip cython3 python3-numpy
-      sudo pip3 install --upgrade --ignore-installed setuptools cython pandas
-    fi
+    
+    sudo apt-get install -y --allow-unauthenticated python3.7 python3-pip python3-numpy
+    sudo pip3 install "Cython>0.24"
+    sudo pip3 install --upgrade --ignore-installed setuptools
+    sudo pip3 install pandas==0.24.2
```

Comment:
For some reason, Linux finds python 3.5 interpreter [here](https://dev.azure.com/mlpack/mlpack/_build/results?buildId=2227&view=logs&j=936136b6-c495-56a6-9e3e-988b38df8fb3&t=75e8b850-1221-57d5-1a8e-32c2f4ae7860&l=188) because of that we have to use pandas v0.24.2. If we don't force this version we can't build the python_bindings. @zoq, I am not sure where the python 3.5 is coming from. When you get a chance could you please take a look at this.
Regards.

Reply:
Do you think you could set the direct path to python 3.7 in the CMake configuration?  `-DPYTHON_EXECUTABLE=/opt/hostedtoolcache/Python/3.7.7/x64/bin/python3`?  (Or something like that.)

Reply:
Hey @rcurtin, I tried a couple of things which include  : 
1. Updating python 3.5 to 3.7 , Without specifying executable using python config auto update.
2. Install using apt update.
3. Moving 3.7 files to python3.
4. Specifying exec path with all the above conditions. This causes results in Cython not found.
5. Changed cmake to first find python3.7 then try for other version.
5. Just specifying executable to python3.7 causes Cython not found.
The only that works is using python3.5 (installed ver) and then specifying to the last pandas version. Could you give me some more insight here. Thanks a lot.